cw721-timelock/src/contract.rs

```diff
@@ -244,3 +247,20 @@ fn query_nft_details(
         recipient: timelock.recipient,
     })
 }
+
+fn query_is_locked(
+    deps: Deps,
+    env: Env,
+    cw721_contract: AndrAddr,
+    token_id: String,
+) -> Result<IsLockedResponse, ContractError> {
+    let lock_id = format!("{}:{}", cw721_contract.into_string(), token_id);
```

Comment:
`get_raw_address` should be used because `cw721_contract ` could be a path instead of a valid address.

Reply:
Make sense.
So I solved this problem.